taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -55,14 +55,11 @@ export default function findLinksAndSetTargets(options) {
         }
         return shouldForce;
       });
-      // FAQ: I am literally double-checking, because I don't trust JavaScript
-      const isExternal = (link.origin !== document.location.origin);
-      const isInternal = (link.host === document.location.host);
-      const shouldSetTarget = shouldForceSetTarget || (
-          ! isInternal && isExternal && ! isMailto
-      );
 
-      if ( shouldSetTarget ) {  
+      const isInternalLink = link.host === baseDocumentHost || link.host === baseDocumentHostWithWWW
+
+      //shouldForceSetTarget could be used in this if statement if passing in options.
+      if (!isInternalLink || isMailto ) {    
```

Comment:
Yup. (Thanks for the comment/reminder.) We avoid `shouldForceSetTarget` for now, cuz it causes the bug.

I'll check that no client needs that feature, and if one does, how to not require this script to handle that.

Reply:
**Update:** No client needs `shouldForceSetTarget`. **Celebrate with Fire:** You may burn all code and variables relevant to it e.g. `pathsToExernalSite`, `pathsToForceSetTarget`. **Bonus:** You may clean up cruft code i.e. perform [CMD-87](https://tacc-main.atlassian.net/browse/CMD-87) at your leisure, now or later.

Reply:
Fixed. [72e3d15](https://github.com/TACC/Core-CMS/pull/805/commits/72e3d15847353f91b0e63751a36cc78bff7a8389)